app/src/main/java/org/thoughtcrime/securesms/permissions/Permissions.java

```diff
@@ -353,12 +354,14 @@ public void run() {
       Context context = this.context.get();
 
       if (context != null) {
-        new AlertDialog.Builder(context, R.style.ThemeOverlay_Session_AlertDialog)
+        AlertDialog alertDialog = new AlertDialog.Builder(context, R.style.ThemeOverlay_Session_AlertDialog)
             .setTitle(R.string.Permissions_permission_required)
             .setMessage(message)
             .setPositiveButton(R.string.Permissions_continue, (dialog, which) -> context.startActivity(getApplicationSettingsIntent(context)))
             .setNegativeButton(android.R.string.cancel, null)
-            .show();
+            .create();
+        alertDialog.getButton(DialogInterface.BUTTON_POSITIVE).setContentDescription(context.getString(R.string.AccessibilityId_continue));
```

Comment:
This one may crash the same as the conversation menu helper for calls

Reply:
Should I delete?

Reply:
might have to replace the line at 363 with these lines:
```java
Button positiveButton = alertDIalog.getButton(DialogInterface.BUTTON_POSITIVE);
if (positiveButton != null) {
    positiveButton.setContentDescription(context.getString(R.string.AccessibilityId_continue));
}
```

may also need to import `Button` with cmd+enter if we aren't using it anywhere else in the file